frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -18,6 +18,10 @@ export const eventUsageLogic = kea<eventUsageLogicType>({
         reportBookmarkletDragged: () => true,
         reportIngestionBookmarkletCollapsible: (activePanels) => ({ activePanels }),
         reportProjectCreationSubmitted: (projectCount, nameLength) => ({ projectCount, nameLength }),
+        reportDemoWarningDismissed: (key) => ({ key }),
```

Comment:
This file is typed wrong. If you look at the generated type.ts file you see that all actions have anys in the type.

Add typing here (e.g. key: string) and you can remove it below in listeners